### ISSUE TITLE
Etcher running in the background CPU at 92%  (Battery broke too, maybe related)

### ISSUE THREAD

**Issue by wwwhtml** [resin-io/etcher#1944]
A few days ago Etcher was downloaded and used once. After that time I quit the application. 

Today it was deleted when it was noticed it was running in the background with a CPU load of about 92%. 

Etcher still the Recycle Bin if any of its files are of your interest.

<img width="750" alt="etcher-cpu-92percent--charger-gone-bad-broken" src="https://user-images.githubusercontent.com/10582407/34512072-b50cf078-f014-11e7-9f3f-11f52806f12c.png">

- **Etcher version: 1.2.1**
- **Operating system and architecture: MacOS Sierra 10.12.6**
- **Image flashed: Rasbpian**
- **Do you see any meaningful error information in the DevTools?**

**Comment by jhermsmeier** [comment]
We've had instances of this before; but this should be gone once https://github.com/resin-io/etcher/pull/1945 is merged

**Comment by jhermsmeier** [comment]
Duplicate of #1288

**Comment by wwwhtml** [comment]
Thank you folks!!
